frontend/src/constants.js

```diff
@@ -85,4 +85,30 @@ export default Object.freeze({
     SHORT_DISTRIBUTION: { text: "Circuit-court" },
     LOCAL: { text: "Local" },
   },
+  Jobs: [
+    {
+      text: "Gestionnaire d'établissement",
+      value: "ESTABLISHMENT_MANAGER",
+    },
+    {
+      text: "Direction Achat Société de restauration",
```

Comment:
```suggestion
      text: "Direction achat société de restauration",
```
Juste pour de la cohérence par rapport aux majuscules et minuscules. Je ne pense pas que ça soit nécessaire de le faire dans le backend aussi.